Source/Server/Startup.cs

```diff
@@ -40,14 +45,23 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
+                app.UseSwagger();
```

Comment:
I would suggest having swagger on always?  As you don't then need to set the runtime up for development and making the APIs more accessible to everyone.

Reply:
Sure, that we can do.